### PR TITLE
Update asr proto for offline diarization

### PR DESCRIPTION
**Pull request by mohnishparmar**

### CODE REVIEW

**Comment by mohnishparmar** [comment]
@ryanleary @findkim please review these proto changes for diarization

File: riva/proto/riva_asr.proto [hunk 1]
```diff
@@ -170,6 +175,23 @@ message StreamingRecognitionConfig {
...
+  bool enable_speaker_diarization = 1;
+
+  // Minimum number of speakers in the conversation. This range gives you more
+  // flexibility by allowing the system to automatically determine the correct
+  // number of speakers. If not set, the default value is 2.
+  int32 min_speaker_count = 2;
+
+  // Maximum number of speakers in the conversation. This range gives you more
+  // flexibility by allowing the system to automatically determine the correct
+  // number of speakers. If not set, the default value is 6.
```
[the remark this answers is not in the document]
Comment (mohnishparmar) [reply riva/proto/riva_asr.proto:192]: It not the default value but the parameter index, the default value should be handled in the code.

File: riva/proto/riva_asr.proto [hunk 2]
```diff
@@ -252,6 +274,13 @@ message WordInfo {
 
   // The word corresponding to this set of information.
   string word = 3;
+
+  // Output only. A distinct integer value is assigned for every speaker within
```
Comment (messiaen) [inline riva/proto/riva_asr.proto:278]: Is it worth mentioning that `0` means no speaker tag?

File: riva/proto/riva_asr.proto [hunk 3]
```diff
@@ -170,6 +175,23 @@ message StreamingRecognitionConfig {
...
+
+  // Minimum number of speakers in the conversation. This range gives you more
+  // flexibility by allowing the system to automatically determine the correct
+  // number of speakers. If not set, the default value is 2.
+  int32 min_speaker_count = 2;
+
+  // Maximum number of speakers in the conversation. This range gives you more
+  // flexibility by allowing the system to automatically determine the correct
+  // number of speakers. If not set, the default value is 6.
+  int32 max_speaker_count = 3;
```
Comment (messiaen) [inline riva/proto/riva_asr.proto:193]: What do `max_speaker_count` and `min_speaker_count` map to in the nemo code? `max_speaker_count` is clear.  What does `min_speaker_count` do?

**Review by messiaen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**